packages/trace-viewer/src/ui/workbench.tsx

```diff
@@ -172,7 +172,7 @@ export const Workbench: React.FunctionComponent<{
     id: 'network',
     title: 'Network',
     count: networkModel.resources.length,
-    render: () => <NetworkTab boundaries={boundaries} networkModel={networkModel} onEntryHovered={setHighlightedEntry}/>
+    render: () => <NetworkTab boundaries={boundaries} networkModel={networkModel} model={model} onEntryHovered={setHighlightedEntry}/>
```

Comment:
Can we avoid passing `model`? Perhaps calculate all the ids beforehand and put them into `NetworkModel` for better separation of concerns?